practice_projects/#Homework pt.py

```diff
@@ -0,0 +1,19 @@
+#Homework pt.1
+
+friend = ("Software is my friend!")
+print (friend)
+
+#Homework pt.2
+
+long_string = "This is a super long string.\n  When I print this to the window, it will continue on the same line\n  unless we do something about it.\n\nThis is for the homework, I would like it to look fancy, please!\n"
```

Comment:
Missing tabs: see https://beginner-software-documentation.atlassian.net/wiki/spaces/GS/pages/3637250/Strings, "Tabs in Strings: section.
Otherwise, the newlines look correct.

Reply:
Figured it out! Thanks